### DISCUSSION TITLE
Discord Bot Problem

### DISCUSSION THREAD

**Discussion by thundernoggin** [snapshot-labs/snapshot#3175] [Q&A]
I'm having a weird problem with the Snapshot bot. I've doubled checked all my permissions. If I use the command /ping it responds as expected. /help on the other hand gives me the error message "the application did not respond". And the !snapshot command doesn't work at all. Any idea what might be happening here?

**Comment by metrox-eth** [comment]
Checking permissions is a good start yep, you have to make sure that the bot has permission to post message in the channel you invoke it. I can't reproduce, for me it works. Maybe you can check again the permissions for this particular channel, if it's still doesn't work try to reinvite the bot.
<img width="414" alt="Capture d’écran 2022-09-23 à 23 59 13" src="https://user-images.githubusercontent.com/55286013/192013994-79d9a460-4b3d-4ec7-bcd0-bb1488a60c0b.png">

**Reply by metrox-eth** [reply]
The command is just / and a menu should popup.

**Reply by thundernoggin** [reply]
Yes thank you. It's peculiar to me that the message menu pops up but only some of the commands work.

**Reply by ChaituVR** [reply]
@thundernoggin Try `/add` command to add a notification. let me know what you see when you run it

**Reply by thundernoggin** [reply]
@ChaituVR The command brings up the dialogue box, channel I know, what is the space dialogue box for? /help still does not work.

**Reply by ChaituVR** [reply]
space is the space you want to subscribe to. example if you want to receive a notification when a new proposal is started in `ens.eth` space, enter `ens.eth`
